helm/vitess/values.yaml

```diff
@@ -229,7 +233,7 @@ vtworker:
 
 # Default values for vttablet resources defined in 'topology'
 vttablet:
-  vitessTag: helm-2.0.1-0
+  vitessTag: helm-2.0.2-0
```

Comment:
While you're in here, maybe add in your proxy flag now that we're bumping the image tag?

Reply:
I hard-coded it in my previous PR here: https://github.com/vitessio/vitess/blob/master/helm/vitess/templates/_vtctld.tpl#L133. You think this should be an option?

Reply:
Nope, my bad for missing the earlier commit. It should definitely just be hardcoded